packages/suite-desktop/electron-builder-config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         '!build/**/*.js.map',
         'dist/**/*.js',
         'package.json',
+        '!**/_postinstall.js', // exclude es5-ext protestware
```

Comment:
I am always thinking whether this can't introduce some sort of false positive match. very likely, something called _postinstall.js will never be relevant part of package, but who knows. 🤔

Reply:
You should really start to think about using something like vite-electron because then you can get rid of this whole blacklist and I am pretty sure it will reduce app size too.

Reply:
I've been looking at electron-vite, but I don't think it solves this problem. It just externalize dependencies automatically, so it can save us 10 lines of configuration we have for it, but other that that I suppose we would face exactly the same issue with it - it would also copy react-native and all it's dependencies if we have it in package.json.

Reply:
That's strange according their docs they are bundling everything, but I did not dig deeply enough maybe.

Reply:
Ultimate solution will be #8842 but for now let's merge this.